vignettes/articles/presentations/technical-demo/technical-demo.qmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Technical Demo"
+subtitle: "Contraceptives DDI Trial Simulation Platform"
+date: 2024-08-27
+date-modified: today
+author: Felix MIL
+institute: |
+  ![](https://esqlabs.com/wp-content/uploads/2018/05/ESQLABS_final_600px.png){height=100}
+embed-resources: true
+format: esqlabs-revealjs
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message = FALSE)
+library(cts)
+```
+
+# 🚩 Introduction
+
+## 👥 Presentations
+
+::: {layout-ncol="3"}
+![Pavel B. - Lead Scientist](img/Pavel.png){style="border-radius: 50%;" width="40%"}
+
+![Felix M. - Project Lead/R Dev.](img/Felix.png){style="border-radius: 50%;" width="40%"}
+
+![Diane L. - Scientist/R Dev](img/Diane.png){style="border-radius: 50%;" width="40%"}
+
+![Rob M. - C# Dev](img/Rob.png){style="border-radius: 50%;" width="40%"}
+
+![Rudolf E. - R Dev](img/Rudolf.png){style="border-radius: 50%;" width="40%"}
+:::
+
+## 🌐 Project Overview
+
+### 🎯 Goal
+
+Develop a clinical trial simulator (**CTS**) for drug-drug interactions (DDI) for contraceptives using physiologically based pharmacokinetic (PBPK) models snapshots.
+
+### 📍 Scope
+
+-   R package
+-   Programming Interface
+
+### 📅 Deadline
+
+31th October 2024
```

Comment:
Mid october

Reply:
Aligned with Laura Kata, it's actually end of October (We shifted this few weeks ago to take into account the already planned vacations). It is updated on Monday too.